server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueIndex.java

```diff
@@ -540,11 +539,13 @@ private void configureStickyFacets(IssueQuery query, SearchOptions options, Map<
       .field(IssueIndexDefinition.FIELD_ISSUE_FUNC_CREATED_AT)
       .dateHistogramInterval(bucketSize)
       .minDocCount(0L)
-      .format(DateUtils.DATETIME_FORMAT)
+      // JodaTime is used to format date times. Timezone with colon is handled with ZZ in JodaTime
+      .format("yyyy-MM-dd'T'HH:mm:ssZZ")
```

Comment:
This update should not be in this commit => this commit is only only updating test code, this is prod code